meetings/Meeting-06-2019/program.md

```diff
@@ -23,16 +23,17 @@ coding and short discussions when necessary.
   + William Hart: **Nemo.jl**
   + William Hart: **AbstractAlgebra.jl**
   + Tommy Hofmann: **Hecke.jl**
-* 17:45: Group photo
-* 18:00: Claus Fieker: **Modules in Oscar - An attempt at a unified structure**
+* 15:00: Coffee Break
+* 15:45: Group photo
+* 16:00: Claus Fieker: **Modules in Oscar - An attempt at a unified structure**
```

Comment:
Claus said that he'll be definitely back in time to give a talk at 16:00 ; but it's not clear to me that he'll be back in time for a photo at 15:45. Perhaps the photo should simply be right after his talk (which according to him will be about ~20 minutes)? Or we move both items 15 minutes to the back? I.e. 16:00 and 16:15?

Reply:
Good point.